### DISCUSSION TITLE
Token Playground Chapter 8, 9 &10: Using the native Stellar Lumens (XLM) using soroban-cli (and in the classic way)

### DISCUSSION THREAD

**Discussion by esteblock** [stellar/sorobounty-spectacular#8] [General]
Hello!

I had a lot of issues trying to interact with the native token inside Soroban. So after researching and asking in Discord, I continued the **[Token Playground series](https://token-playground.gitbook.io/)** with 3 new chapters:

- Chapter 8 : Use the native Stellar Lumens (XLM) the classic way. [See on gitbook](https://token-playground.gitbook.io/guide/index/8_use_xlm_native_inside_classic). [See on dev.to](https://dev.to/esteblock/token-playground-8-use-the-native-stellar-lumens-xlm-the-classic-way-2mlf)
- Chapter 9 : Read the native token (XLM) using soroban-cli. [See on gitbook](https://token-playground.gitbook.io/guide/index/9_read_native_soroban_cli). [See on dev.to](https://dev.to/esteblock/token-playground-chapter-9-read-the-native-token-xlm-using-soroban-cli-cb3)
- Chapter 10: Write the native token (XLM) contract using soroban-cli [See on gitbook](https://token-playground.gitbook.io/guide/index/10_native_xlm_transfer_transfer_from_soroban_cli). [See on dev.to](https://dev.to/esteblock/token-playground-10-write-the-native-token-xlm-contract-using-soroban-cli-1aa6)

In order to do this I wrote two tutorials and some code available [here](https://github.com/esteblock/token-playground/tree/main/src/chapter8), [here](https://github.com/esteblock/token-playground/blob/main/src/chapter9/native_XLM_soroban_read.sh) and [here](https://github.com/esteblock/token-playground/blob/main/src/chapter10/sending_native_XLM_soroban.sh)

Thanks!
__

Check the whole series!

- [Token Playground Gitbook](https://token-playground.gitbook.io/)
- [Token Playground Sereis on dev.to](https://dev.to/esteblock/series/22329)

**Comment by briwylde08** [comment]
Hello! Thank you so much for your Sorobounty Spectacular submission! We will review it and get back to you within a week. 🎉

Best,
Bri

**Comment by esteblock** [comment]
Updated it with one more chapter! (chapter 10)

**Comment by rice2000** [comment]
Amazing!  Thanks for this!

**Comment by briwylde08** [comment]
Hello! We have reviewed your submission. [🎉](https://emojipedia.org/party-popper/) 

We love that it's part of a larger course but would have liked to see a video walkthrough or some sort of playground dapp for calling the token methods in a UI.

Given our guidelines, your submission may qualify for an award of $250 worth of XLM. To receive that award, please fill out [this form](https://stellarform.typeform.com/to/HBjEyCJt) completely and accurately.

Thanks again! Please let us know if you have any questions.

Best,
Bri

**Reply by esteblock** [reply]
Form filled!

**Comment by esteblock** [comment]
Thanks! Dapp would have been great. I agree but time as short while also working on Soroswap deliverables 😁
Thanks!

**Comment by briwylde08** [comment]
Awarded per: https://github.com/stellar/sorobounty-spectacular/discussions/17
